templates/module/src/Entity/Bundle/node.type.yml.twig

```diff
@@ -0,0 +1,17 @@
+langcode: en
+status: true
+dependencies:
+    module:
+        - menu_ui
+third_party_settings:
+    menu_ui:
+        available_menus:
+            - main
+        parent: 'main:'
+name: {{ bundle_title }}
+type: {{ bundle_name }}
+description: 'Generated by Drupal Console'
```

Comment:
Are quotes usually used in yml files? I seem to recall they weren't used. Please educate me.